doc/source/groups_of_atoms.rst

```diff
@@ -208,4 +208,4 @@ and see which fragments are associated with atoms in a smaller :class:`~MDAnalys
 
 .. note::
 
-    :attr:`AtomGroup.fragments <MDAnalysis.core.groups.AtomGroup.fragments>` returns a tuple of fragments with at least one :class:`~MDAnalysis.core.groups.Atom` in the :class:`~MDAnalysis.core.groups.AtomGroup`, not a tuple of fragments where *all* Atoms are in the :class:`~MDAnalysis.core.groups.AtomGroup`.
\ No newline at end of file
+    :attr:`AtomGroup.fragments <MDAnalysis.core.groups.AtomGroup.fragments>` returns a tuple of fragments with at least one :class:`~MDAnalysis.core.groups.Atom` in the :class:`~MDAnalysis.core.groups.AtomGroup`, not a tuple of fragments where *all* Atoms are in the :class:`~MDAnalysis.core.groups.AtomGroup`.
```

Comment:
I might be blind, but i can't see any change here :/

Reply:
Odd that wasn't a file I touched. Maybe an EOL introduction?

Reply:
Yeah looks like empty lines added, github's editor can't remove them either. If it's not breaking things I'm tempted to just say let's keep them?